src/model/urls.ts

```diff
@@ -46,16 +49,18 @@ export const JWT_SUPPORT_URL = `${JWT_URL}/support`;
 // For better minification
 const encode = encodeURIComponent;
 
+export const singleStationUrlV2 = (statId: number) =>
+  `${JWT_V2_ALL_STATIONS_URL}/${encode(String(statId))}`;
 export const singleStationUrl = (statId: number) =>
   `${JWT_ALL_STATIONS_URL}/${encode(String(statId))}`;
 export const feedbackUrl = (type: SupportType) =>
   `${JWT_SUPPORT_URL}/${encode(type)}`;
 export const rentBikeUrl = (statId: number) =>
-  `${singleStationUrl(statId)}/rent`;
+  `${singleStationUrlV2(statId)}/rent`;
 export const reserveBikeUrl = (statId: number) =>
   `${singleStationUrl(statId)}/book`;
```

Comment:
All routes now contain the `/v2/` since the introduction of the locks, EXCEPT the renting 🤦